docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+
+```xml
+<!-- Reference all of WPF -->
+<FrameworkReference Include="Microsoft.WindowsDesktop.App.WPF" />
+```
+
+To use both Windows Forms and WPF APIs, add this reference to the vcxproj:
+
+```xml
+<!-- Reference the entirety of the Windows desktop framework:
+     Windows Forms, WPF, and the types that provide integration between them -->
+<FrameworkReference Include="Microsoft.WindowsDesktop.App" />
+```
+
+Currently, it is not possible to add these references using Visual Studio's reference manager. The project file needs to be updated manually.
+
+## Building without MSBuild
+
+The MSBuild project system makes it easy to port a vcxproj-based project to .NET Core. If you need to build a C++/CLI project for .NET Core directly with cl.exe and link.exe, it can be done with a few additional steps.
+
+1. When compiling, pass `/clr:netcore` to cl.exe.
+2. Reference necessary .NET Core reference assemblies.
+3. When linking, provide the .NET Core app host directory as a LibPath (so that ijwhost.lib can be found).
+4. Copy ijwhost.dll (from the .NET Core app host directory) to the project's output directory.
+5. Make sure a [.runtimeconfig.json](https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md) file exists for the first component of the application that will run managed code. If the application has a managed entry point, this will be created and copied automatically. If the application has a native entry point, though, and a C++/CLI library will be the first time the process uses the .NET Core runtime, you will need to create a runtimeconfig.json file for the library.
+
+## Known issues
+
+There are a couple known issues to look out for when working with C++/CLI projects targeting .NET Core.
+
+1. If the application has a native entry point, the C++/CLI library that first executes managed code needs a [.runtimeconfig.json](https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md) file to be used when the .NET Core runtime is started. C++/CLI projects don't create .runtimeconfig.json files automatically at build time yet, so the file must be generated manually. If a C++/CLI library is called from a managed entry point (which will have its own .runtimeconfig.json file), then the C++/CLI library doesn't need a .runtimeconfig.json file of its own. A simple sample .runtimeconfig.json file looks like this (see the [spec on GitHub](https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md) for more details):
+    ```json
+    {
+          "runtimeOptions": {
+             "tfm": "netcoreapp3.1",
+             "framework": {
+                "name": "Microsoft.NETCore.App",
+                "version": "3.1.0"
+             }
+          }
+    }
+    ```
+2. A WPF framework reference in C++/CLI projects currently causes some extraneous warnings about being unable to import symbols. These can be safely ignored and should be fixed soon.
```

Comment:
```suggestion
- A WPF framework reference in C++/CLI projects currently causes some extraneous warnings about being unable to import symbols. These can be safely ignored and should be fixed soon.
```
Since these two points aren't part of an ordered procedure, just use bullets.